draft-ietf-quic-transport.md

```diff
@@ -1438,6 +1438,51 @@ actual exponent is one-less than the explicit exponent, and the value represents
 to 0xFFFF.
 
 
+### ACK Frames and Packet Protection
+
+ACK frames that acknowledge protected packets MUST be carried in a packet that
+has an equivalent or greater level of packet protection.
+
+Packets that are protected with 1-RTT keys MUST be acknowledged in packets that
+are also protected with 1-RTT keys.
+
+A packet that is not protected and claims to acknowledge a packet number that
+was sent with packet protection is not valid.  An unprotected packet that
+carries acknowledgments for protected packets MUST be discarded in its entirety.
```

Comment:
That was the only thing that made me hesitate to even write this down.  Let's see how things go.  If people find themselves violating this rule, then we will need to come up with some better rules.  In practice, I think that the only thing that needs to happen is that the STREAM frames don't get passed along to the next layer.  That should be easy-ish to ensure.